lib/disposable.dart

```diff
@@ -37,13 +37,25 @@ class _InternalDisposable implements _Disposable {
   }
 }
 
+/// An interface to abstract management from actual disposal machinery.
```

Comment:
A manager for disposable members. The consumer of this interface has precise control over how their disposable members are disposed.